src/EnergyPlus/DataZoneEquipment.hh

```diff
@@ -94,6 +94,7 @@ namespace DataZoneEquipment {
 	extern int const VRFTerminalUnit_Num;
 	extern int const PurchasedAir_Num;
 	extern int const ZoneEvaporativeCoolerUnit_Num; // #13, last zone equipment type to use zone availability manager. The above list must not change or NumValidSysAvailZoneComponents(13) must also change.
+	extern int const ZoneHybridEvaporativeCooler_Num;// #14, last zone equipment type to use zone availability manager. The above list must not change or NumValidSysAvailZoneComponents(14) must also change.
```

Comment:
You copy/pasted the line above, but the comment needs to be removed from the line above or this doesn't make sense.

Reply:
fixed